stor/utils.py

```diff
@@ -197,24 +182,12 @@ def is_filesystem_path(p):
     Returns:
         bool: True if p is a Windows path, False otherwise.
     """
+    from stor_swift.utils import is_swift_path
```

Comment:
Yep, I didn't forget this. I hoisted `SwiftPath` etc here https://github.com/anujkumar93/stor/commit/687b2c89d5d4dbb1d5a548344e8d1353c0b69f91 , which will be part of follow up PR. Since many files have moved, I can't cherry-pick this commit cleanly. Rather than doing the same changes again, I left a comment in that PR, and will pick this up and complete in that PR.